Scripts/Utils/HelpTextFormatter.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * PHPCSDevTools, tools for PHP_CodeSniffer sniff developers.
+ *
+ * @package   PHPCSDevTools
+ * @copyright 2019 PHPCSDevTools Contributors
+ * @license   https://opensource.org/licenses/LGPL-3.0 LGPL3
+ * @link      https://github.com/PHPCSStandards/PHPCSDevTools
+ */
+
+namespace PHPCSDevTools\Scripts\Utils;
+
+/**
+ * Helper class for formatting help text.
+ *
+ * ---------------------------------------------------------------------------------------------
+ * This class is not part of the public API. Backward compatibility is not guaranteed.
+ * ---------------------------------------------------------------------------------------------
+ *
+ * @since 2.0.0
+ */
+class HelpTextFormatter
+{
+    /**
+     * Max width for help text.
+     *
+     * @var int
+     */
+    const MAX_WIDTH = 80;
+
+    /**
+     * Margin for help options.
+     *
+     * @var string
+     */
+    const LEFT_MARGIN = '  ';
+
+    /**
+     * Format help text from a structured array of options.
+     *
+     * @param array<string, array<array<string, string>>> $helpTexts   The help texts to format.
```

Comment:
As the formatter is now in a separate class, I think the expected format for the `$helpTexts` array needs to be documented in more detail.

The class docblock might even be a better place than this function docblock.